psi4/src/psi4/dfocc/manager.cc

```diff
@@ -192,8 +192,15 @@ void DFOCC::omp2_manager() {
     Process::environment.globals["SOS-MP2 CORRELATION ENERGY"] = Esosmp2 - Escf;
     Process::environment.globals["SCSN-MP2 CORRELATION ENERGY"] = Escsnmp2 - Escf;
 
-    variables_["MP2 OPPOSITE-SPIN CORRELATION ENERGY"] = Emp2AB;
-    variables_["MP2 SAME-SPIN CORRELATION ENERGY"] = Emp2AA + Emp2BB;
+    if (reference_ == "UNRESTRICTED") {
```

Comment:
Just checking, but does this cover UHF/ROHF/CUHF references? As a tanged do we have CUHF restrictions?

Reply:
In occ/dfocc, `RESTRICTED` is RHF/RKS and `UNRESTRICTED` is UHF/UKS/ROHF. I don't defend it. I haven't explored CUHF. Maybe another pass -- for this one I'm interested in potentially matching other programs.